shared/js/smart-screen/key_navigation_adapter.js

```diff
@@ -22,6 +28,7 @@
       switch (evt.type) {
         case 'keydown':
         case 'keyup':
+        case this._customevent:
```

Comment:
If we just put customevent falling through. We may have all events without prefix, like keyup. Is this what you want? I would suggest to let this module be aware of `mozbrowserbeforekeydown` and `mozbrowserbeforekeyup`, and use one flag to turn this on or off. Currently, we have mozbrowser(before|after)key(down|up) events. We should have the ability to separate them out. All users of this module should decide which action they want to do while processing different events.